src/www/messages/en.json

```diff
@@ -72,7 +72,9 @@
   "server-added": "Added server “{serverName}”!",
   "server-connected": "Connected to “{serverName}”",
   "server-create-your-own": "Don’t have an access key?<br>Create your own at {openLink}our website{closeLink}.",
+  "server-create-your-own-access": "Don’t have an access key?<br>Create your own at {openLink}our website{closeLink} or<br>request an {openLink2}access key{closeLink}.",
```

Comment:
Why are you creating this message instead of reusing `server-create-your-own-zero-state`?

This will block release, because we don't have the new message in other languages.

Reply:
We need to have separate identifiers for the strings to have the ability to trigger different messages on different conditionals (for example we still need the original farsi strings to display on ios.) I added the english even though they'll never be visible so that we would eventually end up in a consistent TC state will all the strings filled out and corresponding. Having them as separate ids but then secretly having some of the english strings be dups seems like a confusing road to go down.

Is there a translation check somewhere in the release scripting this will block? Or is it a block in the TC i18n scripting?

Reply:
I see, the two new messages are linking to Paskoocheh, but in different places!
It's odd that we have two messages though, looks like an inconsistency. May be worth checking with Jeff.

In any case, we need to make sure we never render messages that are not available, so we should use the dom-if approach if we are rendering these messages that are for Farsi only.

Reply:
Yes, my understanding from @daniellacosse is that the two slightly different messages are the result of a partially-finished migration. Given that that's the existing state I decided to also add an alt-access-version for the second one (since the behavior in the client is that the server-add popup hides the zero-screen when initially opening the app.) even though it wasn't included in the mocks. If that migration gets finished one set of messages can go away.